app/views/people/show.html.erb

```diff
@@ -128,10 +127,11 @@
           <%= render "govuk_publishing_components/components/document_list", items: @person.announcements %>
 
           <div class="read-more">
-            <%= link_to t('announcements.view_all'), announcements_path(people: [@person.slug]) %>
+            <%= link_to t('announcements.view_all', locale: :en), announcements_path(people: [@person.slug]) %>
           </div>
         </section>
       <% end %>
     </div>
   </div>
+  <% binding.pry %>
```

Comment:
this line might need to come out 😉

Reply:
Sorry, should've attached a `do no review` tag to this. Since Whitehall is refusing to run locally, I'm having to use integration at the moment - hence the use of pry 😄 Good catch though!